AppImage-package/bundle.sh

```diff
@@ -20,12 +20,17 @@ cd $HOME/$APP/
 wget -q https://github.com/probonopd/AppImages/raw/master/functions.sh -O ./functions.sh
 . ./functions.sh
 
+URLS=$(apt-get install -qq --reinstall --print-uris  python-tk | cut -d "'" -f 2)
+wget $URLS
```

Comment:
Apparently this didn't work. (It was working when I was still using sudo apt-get inside the script.) Now removing python-tk from .travis.yml, let's see if it works then